modules/internal/Bytes.chpl

```diff
@@ -1298,27 +1298,36 @@ module Bytes {
   }
 
   pragma "no doc"
-  proc ==(a: bytes, b: string) : bool {
-    return doEq(a,b);
+  inline proc !=(a: bytes, b: bytes) : bool {
+    return !doEq(a,b);
+  }
+
+  proc comparisonDeprWarn() {
+    compilerWarning("Comparison between bytes/string is deprecated. " +
```

Comment:
How about "Comparison between bytes and string is deprecated."?

Reply:
good point. will change that